apps/client/app/goods-receipts/page.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Link from 'next/link';
+import { GoodsReceipt } from '../interfaces';
+
+const fetchGoodsReceipts = async (): Promise<GoodsReceipt[]> => {
+  try {
+    const res = await fetch('http://localhost:3100/api/goods-receipts', { cache: 'no-cache' });
+    if (!res.ok) {
+      throw new Error('Failed to fetch goods receipts');
+    }
+    return res.json();
+  } catch (error) {
+    console.error('Error fetching goods receipts:', error);
+    return [];
+  }
+};
+
+const GoodsReceiptsPage = async () => {
+  const goodsReceipts = await fetchGoodsReceipts();
+
+  return (
+    <div className="container mx-auto p-4 bg-base-100 shadow-md rounded-lg">
+      <h1 className="text-3xl font-bold mb-4 text-accent-content">Goods Receipts</h1>
+      <table className="table-auto w-full bg-base-100 text-sm rounded-lg shadow-md">
+        <thead>
+          <tr className="bg-accent-content text-base-100">
+            <th className="p-4 text-left">Goods Receipt Number</th>
+            <th className="p-4 text-left">Purchase Order ID</th>
+            <th className="p-4 text-left">Received Date</th>
+            <th className="p-4 text-left">Actions</th>
```

Comment:
lots of repetition here. Tailwind way of thinking is that this should be ok, but it could be easier to read if you turn the values into an object or array and then map over them so this wouldn't be repeated 4 times.

Reply:
I certainly could do that but what are your thoughts on readability? To me this is more readable